benchmarks/maxtext_xpk_runner.py

```diff
@@ -327,11 +338,12 @@ def generate_xpk_workload_cmd(
     libtpu_version: str,
     base_output_directory: str,
     buffer_size: int,
+    num_steps: int = 100,
+    xpk_path: str = '~/xpk',
+    pathways_config: PathwaysConfig = None,
 ):
   """Generates a command to run a maxstar model on XPK."""
-  num_steps = 20
   time.localtime()
-  test_purpose_name = f'maxstar-benchmarks-{model.model_name}-{libtpu_version}'
```

Comment:
can we add back  
```
 if "steps" not in model.tuning_params:
    num_steps = model.tuning_params["steps"]
```
so that workload specified steps will not be overwritten.

Reply:
I'm not quite sure this does what's intended. I believe this code is better. I've added it in.

```
  if not num_steps and 'steps' in model.tuning_params:
    num_steps = model.tuning_params['steps']
  elif not num_steps:
    num_steps = 20
```

Reply:
(Minor Update, ignore the comment above). Made the change when we're doing the tuning in particular.